tests/MongoDB.Driver.Core.Tests/Core/Servers/ServerTests.cs

```diff
@@ -149,6 +149,61 @@ public void Dispose_should_dispose_the_server()
             _capturedEvents.Next().Should().BeOfType<ServerClosedEvent>();
             _capturedEvents.Any().Should().BeFalse();
         }
+        
+        [Theory]
+        [ParameterAttributeData]
+        public void GetChannel_should_clear_connection_pool_when_opening_connection_throws_MongoAuthenticationException(
+            [Values(false, true)] bool async)
+        {      
+            var connectionId = new ConnectionId(new ServerId(_clusterId, _endPoint));
+            var connectionHandle = new Mock<IConnectionHandle>();
```

Comment:
I would name this variable mockConnectionHandle (or even just mockConnection).

Reply:
Fixed!